src/chat.rs

```diff
@@ -2732,6 +2735,57 @@ async fn prepare_send_msg(
     Ok(row_ids)
 }
 
+/// Renders the Message or splits it into Post-Message and Pre-Message.
+///
+/// Pre-Message is a small message with metadata which announces a larger Post-Message.
+/// Post-Messages are not downloaded in the background.
```

Comment:
If the pre-message is missing, shouldn't the "post-message" be downloaded in background (like usual emails)?